Mathlib/Data/Real/EReal.lean

```diff
@@ -1081,6 +1122,17 @@ theorem top_mul_of_pos {x : EReal} (h : 0 < x) : ⊤ * x = ⊤ := by
   exact mul_top_of_pos h
 #align ereal.top_mul_of_pos EReal.top_mul_of_pos
 
+/-- The product of two positive extended real numbers is positive. -/
+@[simp]
```

Comment:
```suggestion
```

Reply:
Oh sorry. I changed it yesterday, but I didn't pushed it 🤦‍♂️.